packages/flutter/lib/src/material/floating_action_button_theme.dart

```diff
@@ -96,6 +101,25 @@ class FloatingActionButtonThemeData with Diagnosticable {
   /// ignored.
   final bool? enableFeedback;
 
+  /// The size constraints for the [FloatingActionButton].
+  final BoxConstraints? sizeConstraints;
+
+  /// The size constraints for the [FloatingActionButton] when using the small
+  /// constructor.
```

Comment:
When we're referring to a specific factory constructor, can't we use [FloatingActionButton.small] (here and elsewhere)?

Reply:
Done